homeassistant/components/wled/config_flow.py

```diff
@@ -10,11 +10,13 @@
 from homeassistant.config_entries import ConfigEntry, ConfigFlow, OptionsFlow
 from homeassistant.const import CONF_HOST, CONF_MAC
 from homeassistant.core import callback
-from homeassistant.data_entry_flow import FlowResult
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 
 from .const import CONF_KEEP_MASTER_LIGHT, DEFAULT_KEEP_MASTER_LIGHT, DOMAIN
 
+if TYPE_CHECKING:
```

Comment:
In the config-flow, these three are only imported for TYPE_CHECKING.
- homeassistant.config_entries.ConfigEntry
- homeassistant.components.zeroconf
- wled.Device

Should they also be guarded?